.docker/parameters.yml

```diff
@@ -31,7 +31,13 @@ parameters:
     subject_rewrites: []
     example_rds_articles:
         '16846': https://repro.elifesciences.org/example.html
+        '30274': https://repro.elifesciences.org/example.html
     rds_articles:
+        '30274':
+            date: '2030-01-01T00:00:00Z'
```

Comment:
looking at how https://github.com/elifesciences/builder-configuration/blob/master/pillar/rds-articles.yaml#L13 gets into this file on `journal--*` instance, this should already be transparently available, without `journal-formula` changes needed

Reply:
confirmed:
```
elife@prod--journal:/srv/journal$ grep "\brds_articles" -A 2 parameters.yml 
    rds_articles:
        {"30274": {"date": "2030-01-01T00:00:00Z", "display": "https://stencila.elifesciences.org/elife/30274/main/?elife", "download": "https://stencila.elifesciences.org/elife/30274/files/download/article-30274.md"}}
    
```